README_new.md

```diff
@@ -0,0 +1,617 @@
+service](#As_system_service).
+
+In order to check the status of the broker, use the following command
+with superuser privileges (using the root user or the sudo command):
+
+    /etc/init.d/contextBroker status
+
+If broker is running you will get:
+
+    Checking contextBroker...                         Running
+
+If broker is not running you will get:
+
+    Checking contextBroker...                         Service not running
+
+## API Overview
+
+In order to create an entity (Room1) with two attributes (temperature and pressure):
+
+    (curl <orion_host>:1026/v1/contextEntities/Room1 -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' -X POST -d @- | python -mjson.tool) <<EOF
+    {
+      "attributes" : [
+    {
+      "name" : "temperature",
+      "type" : "float",
+      "value" : "23"
+    },
+    {
+      "name" : "pressure",
+      "type" : "integer",
+      "value" : "720"
+    }
+      ]
+    }
+    EOF
+
+In order to query the entity:
+
+    curl <orion_host>:1026/v1/contextEntities/Room1 -s -S --header 'Accept: application/json' | python -mjson.tool
+
+In order to update one of the entity atributes (temperature):
+
+    (curl <orion_host>:1026/v1/contextEntities/Room2/attributes/temperature -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' -X PUT -d @- | python -mjson.tool) <<EOF
+    {
+       "value" : "26.3"
+    }
+    EOF
+
+Please have a look to the [Quick Start guide](doc/md/quick_start_guide.md) if you want to test these operations in an actual public instance of Oriion Context Broker. In addition, have a look to the API Walkthrough and API Reference sections below in order to know more details about the API (subscriptions, registrations, etc.).
+
+## API Walkthrough
+
+* [FIWARE NGSI v1](doc/md/user/walkthrough_apiv1.md) (Markdown)
+* [FIWARE NGSI v2](http://telefonicaid.github.io/fiware-orion/api/v2/cookbook) (Apiary) - *ongoing*
+
+## API Reference Documentation
+
+* [FIWARE NGSI v1](http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/FI-WARE_NGSI:_publicly_available_documents) (XSD and PDF)
+* [FIWARE NGSI v2](http://telefonicaid.github.io/fiware-orion/api/v2/) (Apiary) - *ongoing*
+
+## Testing
+
+### Ent-to-end tests
+
+The functional_test makefile target is used for running end-to-end tests:
+
+    make functional_test INSTALL_DIR=~
+
+Please have a look to the section [on building the source code](#from-sources) in order to get more information about how to prepare the environment to run the functional_test target.
+
+### Unit tets
+
+The unit_test makefile target is used for running the unit tests:
+
+    make functional_test INSTALL_DIR=~
+
+Please have a look to the section [on building the source code](#from-sources) in order to get more information about how to prepare the environment to run the unit_test target.
+
+## Advanced topics:
+
+* Installation and aministration
```

Comment:
Topics have been ordered in relevance order (most relevant: up in the list) within each category (admin and API). Any feedback about this priority is welcome.